.github/workflows/python-publish.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       run: |
         python -m build --sdist --wheel --outdir dist/ .
     - name: Publish package
-      uses: pypa/gh-action-pypi-publish@v1.9.0
+      uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I wonder if dependabot recognizes this kind of pattern. Is there any documentation available saying so?

Reply:
It seems that Dependabot only looks at semantic versioning. It is better to use a specific version in this case: https://github.com/eduNEXT/eox-hooks/commit/03c93802058b851dd1b049cdcd41d23b3fb1d1c3